crates/admin/src/rest_api/deployments.rs

```diff
@@ -95,17 +95,12 @@ pub async fn create_deployment<V>(
         ApplyMode::Apply
     };
 
-    let (id, services) = state
-        .task_center
-        .run_in_scope("create-deployment", None, async {
-            log_error(
-                state
-                    .schema_registry
-                    .register_deployment(discover_endpoint, force, apply_mode)
-                    .await,
-            )
-        })
-        .await?;
+    let (id, services) = log_error(
```

Comment:
Just figured the existence of this method `log_error`... I wonder if we can make it a trait and put it in `restate_errors`.

```
trait CodedErrorResultExt {
  fn warn_it(self) -> Self
}

impl<T, E: CodedError> CodedErrorResultExt for Result<T, E> { .. }
```

Reply:
Will add it.